tailwind.config.js

```diff
@@ -202,7 +202,26 @@ module.exports = {
         ring: 'inset 0 0 0 1px white',
         'ring-1.5': 'inset 0 0 0 1.5px white',
       },
+      typography: (theme) => ({
+        DEFAULT: {
+          css: {
+            '--tw-prose-body': theme('colors.dark-charcoal'),
+            '--tw-prose-headings': theme('colors.dark-charcoal'),
+            '--tw-prose-links': theme('colors.pink'),
+            a: {
+              textDecoration: 'none',
+              '&:hover': {
+                'text-decoration': 'underline',
+              },
+            },
+          },
+        },
+      }),
```

Comment:
Isn't this redundant with what we have in `tailwind.css` in the `@base` layer extension?

Reply:
With the new tailwind plugin, typography, the texts and links on the content pages have different classes applied to them, so it becomes necessary.